config/providers.php

```diff
@@ -4,21 +4,52 @@
 
 use App\Provider\CacheProvider;
 use App\Provider\EventDispatcherProvider;
+use App\Provider\FileRotatorProvider;
+use App\Provider\FileTargetProvider;
 use App\Provider\LoggerProvider;
 use App\Provider\MiddlewareProvider;
-use App\Provider\PSR17Provider;
+use App\Provider\Psr17Provider;
 use App\Provider\RouterProvider;
+use App\Provider\SessionProvider;
 use App\Provider\WebViewProvider;
 use Yiisoft\Arrays\Modifier\ReverseBlockMerge;
 
 return [
-    'yiisoft/router' => RouterProvider::class,
-    'yiisoft/psr17' => PSR17Provider::class,
-    'yiisoft/web' => MiddlewareProvider::class,
-    'yiisoft/cache' => CacheProvider::class,
-    'yiisoft/eventDispatcher' => EventDispatcherProvider::class,
-    'yiisoft/logger' => LoggerProvider::class,
-    'yiisoft/view' => WebViewProvider::class,
+    'yiisoft/app/router' => RouterProvider::class,
```

Comment:
Why is it yiisoft/app/router instead of yiisoft/router?

Reply:
I thought the standard users would be vendor-name/package-name/provider, i think we should move the providers to each package and configure them by parameters.